compiler/src/iree/compiler/Dialect/Stream/Transforms/EncodeTensors.cpp

```diff
@@ -33,6 +34,17 @@ namespace IREE {
 namespace Stream {
 namespace {
 
+//===----------------------------------------------------------------------===//
+// Bitwidth utilities
+//===----------------------------------------------------------------------===//
+static int64_t getTypeBitwidth(Type type) {
```

Comment:
Instead of replicating this we should centralize it to where we currently have the getRoundedElementByteWidth - a getElementBitWidth helper alongside it in https://github.com/openxla/iree/blob/main/compiler/src/iree/compiler/Dialect/Util/IR/UtilTypes.h#L204 would let us have one place to carry this logic.

Reply:
Fixed.